src/Root.tsx

```diff
@@ -0,0 +1,25 @@
+import {
+  HashRouter as Router,
+  Routes,
+  Route,
+  Navigate,
+} from 'react-router-dom';
+import { App } from './App';
+import { HomePage } from './components/HomePage/HomePage';
+import { PeoplePage } from './components/PeoplePage/PeoplePage';
+import { PageNotFound } from './components/PageNotFound/PageNotFound';
+
+export const Root = () => (
+  <Router>
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route index element={<HomePage />} />
+        <Route path="home" element={<Navigate to="/" replace />} />
+        <Route path="people" element={<PeoplePage />}>
+          <Route path=":slugId" element={<PeoplePage />} />
```

Comment:
It seems you're trying to render the same component `PeoplePage` for both the '/people' route and the nested '/people/:slugId' route. If `PeoplePage` is capable of handling both cases, you might not need the parent '/people' route. If they're meant to render different components, you should update the '/people/:slugId' route's element.